docs/network-policy.md

```diff
@@ -236,8 +241,83 @@ ephemeral and unpredictable.
 - Rules assume the priority in which they are written. i.e. rule set at top
   takes precedence over a rule set below it.
 
+## Antrea NetworkPolicy
+
+Antrea NetworkPolicy is another Policy CRD, which is similar to the
+ClusterNetworkPolicy CRD, however its scope is limited to a Namespace.
+The purpose of introducing this CRD is to allow admins to take advantage of
```

Comment:
is it still targeted at cluster admins specifically?

I'm asking because above you removed "admin-created" from this sentence:
> Thus, admin created tiered policies have a higher precedence over developer-created K8s NetworkPolicies.

And I assumed it was because of the addition of the Antrea NetworkPolicy CRD.

Reply:
yes i thought about it while adding the ANP.. but then since one can add a ANP with higher precedence over existing CNP.. it does not make sense to have ANP associated with devs.. i will restore the original statement above

Reply:
Agreed that both CRDs should be admin oriented